backend/siarnaq/api/user/views.py

```diff
@@ -75,6 +75,14 @@ def me(self, request):
                 serializer.save()
                 return Response(serializer.data)
 
+    @extend_schema(
+        responses={
+            status.HTTP_200_OK: {
+                "type": "object",
+                "additionalProperties": {"$ref": "#/components/schemas/TeamPublic"},
+            }
```

Comment:
this schema dict describes the response as an object with arbitrary keys with each value conforming to the schema TeamPublic (which is generated from TeamPublicSerializer)